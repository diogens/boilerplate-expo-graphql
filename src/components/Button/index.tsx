import React from 'react'
import * as S from './styles'
import { Text } from 'react-native'

const Button = () => (
  <S.Wrapper>
    <Text>Button</Text>
  </S.Wrapper>
)

export default Button