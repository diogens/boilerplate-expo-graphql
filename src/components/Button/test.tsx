import React from 'react'
import { render, screen } from '@testing-library/react-native'

import Button from '.'

describe('<Button />', () => {
  it('should render the heading', () => {
    const { container, toJSON, getByText, debug } = render(<Button />)

    debug()

    expect(getByText('Button'))

    expect(toJSON()).toMatchSnapshot()
  })
})
