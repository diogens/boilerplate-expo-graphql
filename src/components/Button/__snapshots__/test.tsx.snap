// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should render the heading 1`] = `
<View
  style={
    Array [
      Object {},
    ]
  }
>
  <Text>
    Button
  </Text>
</View>
`;
